src/test/java/uk/nhs/prm/deduction/e2e/queue/QueueMessageHelper.java

```diff
@@ -108,7 +108,7 @@ private List<SqsMessage> findMessagesOnQueue(int visibilityTimeout) {
     private SqsMessage findMessageContaining(String substring) {
         List<SqsMessage>  allMessages = thinlyWrappedSqsClient.readThroughMessages(this.queueUri, 180);
         for (SqsMessage message : allMessages) {
-            log(String.format("just finding message, checking conversationId: %s", this.queueUri));
+            log(String.format("Finding message with substring %s on queue: %s", substring, this.queueUri));
             if (message.contains(substring)) {
                 return message;
             }
```

Comment:
This can be replaced with:

```
    private SqsMessage findMessageContainingWitHigherVisibilityTimeOut(String substring) {
        return thinlyWrappedSqsClient.readThroughMessages(this.queueUri, 600)
                .stream()
                .filter(message -> message.contains(substring))
                .findFirst()
                .orElseThrow(RuntimeException::new);
    }
```